test/specs/reducers/conversation-reducer.spec.js

```diff
@@ -105,13 +112,28 @@ describe('Conversation reducer', () => {
             };
             const afterState = ConversationReducer(beforeState, {
                 type: ADD_MESSAGE,
-                message: MESSAGE_TO_ADD
+                message: MESSAGE_FROM_APP_USER
             });
             afterState.messages.length.should.eq(3);
             afterState.messages[0].should.eql(MESSAGE_1);
-            afterState.messages[1].should.eql(MESSAGE_TO_ADD);
+            afterState.messages[1].should.eql(MESSAGE_FROM_APP_USER);
             afterState.messages[2].should.eql(UPLOADING_IMAGE);
         });
+
+        it('should add appMaker message', () => {
```

Comment:
Can you add these scenarios : multiple messages from the same appMaker and multiple messages from different appMakers ?